src/QMCDrivers/WFOpt/QMCFixedSampleLinearOptimizeBatched.h

```diff
@@ -60,9 +72,10 @@ class QMCFixedSampleLinearOptimizeBatched : public QMCLinearOptimizeBatched,
   ///process xml node value (parameters for both VMC and OPT) for the actual optimization
   bool processOptXML(xmlNodePtr cur, const std::string& vmcMove, bool reportH5, bool useGPU);
 
-  RealType Func(RealType dl) override;
+  RealType costFunc(RealType dl);
 
 private:
+  OptimizationFunction opt_;
```

Comment:
Could you brainstorm a name less confusing than `opt_`?  `optTarget` is confusing already.